service/src/test/node_start.ts

```diff
@@ -0,0 +1,69 @@
+import path from 'path';
+import { fileURLToPath } from 'url';
+import { archLogger } from '../utils/chalk-theme';
+import { TestSuite } from './test_suite';
+import 'dotenv/config';
+import { getWeb3Interface } from '../scripts/web3-interface';
+import { ethers, Wallet } from 'ethers';
+
+export async function runTests() {
+    archLogger.warn("\n\nVerify service starts with expected output");
+
+    const cwd = path.dirname(fileURLToPath(import.meta.url));
+    const opts = { sourceFile: '../index.js', timeout: 1000, restartNetwork: true };
+
+    let testSuite: TestSuite;
+
+    testSuite = new TestSuite(cwd, process.env.TEST_CONTRACTS_DIRECTORY!);
+
+    const cleanup = await testSuite.startLocalNetwork();
+
+    archLogger.warn("\n\n Listens for messages");
+    await testSuite.expectOutput('listening to stream on protocol: /message', { ...opts, restartNetwork: false });
+
+    archLogger.warn("\n\n Shows balances");
+    await testSuite.expectOutput('YOUR BALANCES:', opts);
```

Comment:
Since we may have to run all these tests in a single file, for readability it may help to do large comments to separate the tests, something like:

```
/**
*
* TEST: "YOUR BALANCES" OUTPUT
*
**/
```

or just something to break up where tests are being run in the file

Reply:
Ah yes this will very much help with readability. Thanks